cost/aws/burstable_instance_cloudwatch_credit_utilization/aws_burstable_instance_cloudwatch_credit_utilization.pt

```diff
@@ -0,0 +1,753 @@
+            "cloudwatch_cpu_surplus_credits_charged_avg": result.cloudwatch_cpu_surplus_credits_charged_avg,
+            "cloudwatch_cpu_surplus_credits_charged_min": result.cloudwatch_cpu_surplus_credits_charged_min,
+            "cloudwatch_cpu_surplus_credits_charged_max": result.cloudwatch_cpu_surplus_credits_charged_max,
+            "next_instance_size": result.instance_type_info["down"]
+          }
+        )
+      }
+
+      if ( param_num_cpu_surplus_credits != -1 && (result.cloudwatch_cpu_surplus_credit_balance_min != "10000") && (result.cloudwatch_cpu_surplus_credit_balance_min >= param_num_cpu_surplus_credits)){
+        merged_results.push(
+          {
+            "region": result.region,
+            "instanceId": result.instanceId,
+            "instanceType": result.instanceType,
+            "instance_type_info": result.instance_type_info,
+            "privateDnsName": result.privateDnsName,
+            "hostname": result.hostname,
+            "tag_set": result.tag_set,
+            "cpu_minimum": result.cpu_minimum,
+            "cpu_maximum": result.cpu_maximum,
+            "cpu_average": result.cpu_average,
+            "cloudwatch_cpu_credit_balance_avg": result.cloudwatch_cpu_credit_balance_avg,
+            "cloudwatch_cpu_credit_balance_max": result.cloudwatch_cpu_credit_balance_max,
+            "cloudwatch_cpu_credit_balance_min": result.cloudwatch_cpu_credit_balance_min,
+            "cloudwatch_cpu_surplus_credit_balance_avg": result.cloudwatch_cpu_surplus_credit_balance_avg,
+            "cloudwatch_cpu_surplus_credit_balance_max": result.cloudwatch_cpu_surplus_credit_balance_max,
+            "cloudwatch_cpu_surplus_credit_balance_min": result.cloudwatch_cpu_surplus_credit_balance_min,
+            "cloudwatch_cpu_surplus_credits_charged_avg": result.cloudwatch_cpu_surplus_credits_charged_avg,
+            "cloudwatch_cpu_surplus_credits_charged_min": result.cloudwatch_cpu_surplus_credits_charged_min,
+            "cloudwatch_cpu_surplus_credits_charged_max": result.cloudwatch_cpu_surplus_credits_charged_max,
+            "next_instance_size": result.instance_type_info["up"]
+          }
+        )
+      }
+    })
+  EOS
+end
+
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_utilization" do
+  validate_each $ds_check_merged_metrics do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows containing AWS instance CloudWatch utilization data"
+    detail_template <<-EOS
+# AWS Instance Utilization
+| Region | instanceID | Instance Type | Hostname | Private DNS Name | CPU Maximum % | CPU Average % | CPU Credit Balance Minimum | CPU Surplus Credit Balance Minimum | CPU Surplus Credit Charged Maximum | Max Earnable Credits | Next Instance |
```

Comment:
I guess it's the "next instance" that indicates whether this is being sized up or down? That wasn't 100% clear to me - maybe try to make that more clear somehow